### PR TITLE
add assertion on NSTemplateTier.spec.clusterResource

### PR DESCRIPTION
**Pull request by xcoulon**
e2e tests for https://github.com/codeready-toolchain/host-operator/pull/161

### CODE REVIEW

File: test/e2e/nstemplatetier-basic.yaml [hunk 1]
```diff
@@ -75,4 +75,32 @@ spec:
...
+    type: stage
+  clusterResources:
+    revision: 000000a
+    template:
+      apiVersion: template.openshift.io/v1
+      kind: Template
+      metadata:
+        labels:
+          toolchain.dev.openshift.com/provider: codeready-toolchain
+        name: advanced-cluster-resources
```
Comment (MatousJobanek) [inline test/e2e/nstemplatetier-basic.yaml]: this is basic tier, not advanced:
```suggestion
        name: basic-cluster-resources
```

Comment (xcoulon) [reply test/e2e/nstemplatetier-basic.yaml]: ah yes, good catch! 🤦‍♂ 
thanks!

**Pull request merged**